ibutex.py

```diff
@@ -63,6 +66,11 @@
 
 if not args.quick:
 	subprocess.call(fullbib)
+	if args.sagemath:
+		success = subprocess.call(fullsage)
+		if success != 0:
+			print("error running sagetex, aborting tex build! …")
```

Comment:
final remarks: please prepend ":: " to the error message to keep it consistent with the compiling error above